x/wasm/types/proposal.pb.go

```diff
@@ -44,18 +40,23 @@ type StoreCodeProposal struct {
 	InstantiatePermission *AccessConfig `protobuf:"bytes,7,opt,name=instantiate_permission,json=instantiatePermission,proto3" json:"instantiate_permission,omitempty"`
 	// UnpinCode code on upload, optional
 	UnpinCode bool `protobuf:"varint,8,opt,name=unpin_code,json=unpinCode,proto3" json:"unpin_code,omitempty"`
+	// Source is the URL where the code is hosted
+	Source string `protobuf:"bytes,9,opt,name=source,proto3" json:"source,omitempty"`
+	// Builder is the docker image used to build the code deterministically, used for smart
+	// contract verification
+	Builder string `protobuf:"bytes,10,opt,name=builder,proto3" json:"builder,omitempty"`
+	// CodeHash is the SHA256 sum of the code outputted by builder, used for smart contract verification
+	CodeHash []byte `protobuf:"bytes,11,opt,name=code_hash,json=codeHash,proto3" json:"code_hash,omitempty"`
 }
 
 func (m *StoreCodeProposal) Reset()      { *m = StoreCodeProposal{} }
 func (*StoreCodeProposal) ProtoMessage() {}
 func (*StoreCodeProposal) Descriptor() ([]byte, []int) {
 	return fileDescriptor_be6422d717c730cb, []int{0}
 }
-
```

Comment:
I've used `make proto-gen` to generate proto code. All proto files had these formatting changes, reverted them.

Reply:
Please use `make proto-all` which also runs the formatter (`make format`)